libnmstate/nm/ipv6.py

```diff
@@ -158,3 +163,33 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip6_config()
     return None
+
+
+def get_route_running():
```

Comment:
IMHO it would be more natural to use `get_running_routes()` instead of reversing the words. Also for `get_configured_routes()`.